src/blockchain_txn_poc_receipts_v1.proto

```diff
@@ -2,8 +2,8 @@ syntax = "proto3";
 package helium;
 
 enum origin {
-  p2p = 0;
-  radio = 1;
```

Comment:
Please don’t rename fields

Reply:
This was done due to the following error:

> radio.proto:3:9: "helium.radio" is already defined (as something other than a package) in file "blockchain_txn_poc_receipts_v1.proto".

This appears to be an issue with protobuf namespacing, I'm not sure why things that aren't `protoc` didn't care and I made this change based on a recommendation in an SO answer from one of protobuf's original authors about how to handle these conflicts. If there is an alternate method that wouldn't break the naming that'd be preferable, but I am unaware of what that would be.

Reply:
The problem is that it'll break existing erlang and rust users of this message.

Reply:
Will try to dig into those a little bit more to compare, thanks for the tip.